### PR TITLE
Add plotly_express

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
@conda-forge/help-python This is ready for review

File: recipes/plotly_express/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ff73a41ce02fb43d1d8e8fa131ef3e6589857349ca216b941b8f3f862bce0278
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```
Comment (xhochy) [inline recipes/plotly_express/meta.yaml]: No need for these arguments anymore, they are included by default.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/plotly_express/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - numpy >=1.11
```
Comment (xhochy) [inline recipes/plotly_express/meta.yaml]: Are all these dependencies really required for building? I guess they are only needed in `run`.

**Comment by dpryan79** [comment]
The last time I took them out I was asked to put them all back in :p

**Comment by xhochy** [comment]
> The last time I took them out I was asked to put them all back in :p
> […](#)
> Sent from my iPhone
> On 16. Sep 2019, at 11:07, Uwe L. Korn ***@***.***> wrote: @xhochy commented on this pull request. In recipes/plotly_express/meta.yaml: > @@ -0,0 +1,49 @@ +{% set name = "plotly_express" %} +{% set version = "0.4.1" %} + +package: + name: "{{ name|lower }}" + version: "{{ version }}" + +source: + url: "[https://pypi.io/packages/source/{{](https://pypi.io/packages/source/%7B%7B) name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz" + sha256: ff73a41ce02fb43d1d8e8fa131ef3e6589857349ca216b941b8f3f862bce0278 + +build: + number: 0 + noarch: python + script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv" No need for these arguments anymore, they are included by default. ⬇️ Suggested change - script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv" + script: "{{ PYTHON }} -m pip install . -vv" In recipes/plotly_express/meta.yaml: > +package: + name: "{{ name|lower }}" + version: "{{ version }}" + +source: + url: "[https://pypi.io/packages/source/{{](https://pypi.io/packages/source/%7B%7B) name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz" + sha256: ff73a41ce02fb43d1d8e8fa131ef3e6589857349ca216b941b8f3f862bce0278 + +build: + number: 0 + noarch: python + script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv" + +requirements: + host: + - numpy >=1.11 Are all these dependencies really required for building? I guess they are only needed in run. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

That's weird, we normally once merge recipes that only have the dependencies in there that are required for the `setup.py` to run.

File: recipes/plotly_express/meta.yaml (hunk 2 above)
Comment (dpryan79) [reply recipes/plotly_express/meta.yaml]: It's hit or miss whether people write setup.py to work properly without all of the dependencies (and without specifying `--no-deps`), but let's see.

**Comment by scopatz** [comment]
Thanks!

**Comment by dpryan79** [comment]
Thanks! One of these days I'll actually remember the preferred conda-forge way of handling python packages :P

**Pull request merged**